### DISCUSSION TITLE
Auto format f Strings => remove f on {} removal

### DISCUSSION THREAD

**Discussion by IGLADI** [microsoft/pylance-release#5525] [Enhancement]
![image](https://github.com/microsoft/pylance-release/assets/128155180/01bdcfd4-fe4c-4f04-bb6c-ca2a0f51d95f)
I really like this feature, but it's actually one way only instead of two ways, would be great if it also delete the f if you remove all {}

This it wat it currently does when the feature is activated:
![image](https://github.com/microsoft/pylance-release/assets/128155180/3acc7f51-efcc-4244-8fd0-6df3aaa19a08)
now I type {}
![image](https://github.com/microsoft/pylance-release/assets/128155180/2d8003c3-d4df-48e0-8a6e-761461bd3848)
and now I remove {}
![image](https://github.com/microsoft/pylance-release/assets/128155180/e06754a3-cda5-4a7f-99c3-626f1938a196)
Which let's you with ugly fstr as they should be str (most linters also encourage this as you see per the line turning red)

What should happen at last change imo:
![image](https://github.com/microsoft/pylance-release/assets/128155180/12845baa-a0cb-4c55-af67-b7eb40f05ebb)

**Comment by rchiodo** [comment]
Sounds like a cool idea to me too. If the string has no `{` in it, remove the f on the front.
